### PR TITLE
(Chore) Review and fix flow-types

### PR DESCRIPTION
**Pull request by fernandomg**
[#163621735](https://www.pivotaltracker.com/story/show/163621735)

Add flow-type and flow-typed defs.

Fixed flow errors.

To check for further errors run: `npm run flow`

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -3,10 +3,26 @@
...
 ./scripts/.*
-./node_modules/.*
+./ossl/.*
+./radata/.*
+.*/node_modules/.*
 
 [libs]
-./flow-typed/.*
\ No newline at end of file
+./flow-typed/.*
```
Comment (dminones) [inline .flowconfig]: I think we should remove this from the ignore. https://stackoverflow.com/questions/54327236/should-i-keep-flow-typed-npm-folder-versionated-in-git

Comment (fernandomg) [reply .flowconfig]: This is under `[libs]` so `flow` can lookup for types definitions inside `flow-typed` folder.

Also I added the folder to the repo, so we can version it. And it's not listed in the `.gitignore` file either.

**Comment by fernandomg** [comment]
After solving `master` merge conflicts, had to fix new flow-type errors.

Please @dminones, @sirpy give it a look whenever you can.

Thanks!

File: .flowconfig (hunk 1 above)
Comment (dminones) [reply .flowconfig]: You're right. My bad

File: src/server/blockchain/blockchain-types.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ export interface Subscribe<T> {
         id: string;
         subscribe(callback?: Callback<Subscribe<T>>): Subscribe<T>;
         unsubscribe(callback?: Callback<boolean>): void | boolean;
-        arguments: object;
+        arguments: any;
```
Comment (dminones) [inline src/server/blockchain/blockchain-types.js]: Maybe `{}` works better in these cases? 
https://flow.org/en/docs/types/objects/#toc-unsealed-objects

Comment (dminones) [inline src/server/blockchain/blockchain-types.js]:
```suggestion
        arguments: {};
```

**Pull request merged**